docs/release.md

```diff
@@ -0,0 +1,21 @@
+# Application Release Process
+
+When merging to the `develop` branch, the CIIP application is continuously deployed to our development environment at https://dev.ciip.gov.bc.ca/. This document describes the steps required to deploying the application to our test (QA) and production environments.
+
+## 1 Release Pull Request
+
+- 1.1 Create a chore/release branch
+- 1.2 From the root of the repository, run `./app/node_modules/.bin/standard-version`. This will generate the [changelog](../CHANGELOG.md), increase the version number following [semantic versioning](https://semver.org/), commit the changes and tag that commit.
+- 1.3 note the version number printed by standard-version (referred to as &`<version>` below)
+- 1.4 run `cd schema/ && sqitch tag v<version>  -m "release v<version>"`
+- 1.5 In helm/cas-ciip-portal/Chart.yaml, set `appVersion` to `<version>`, and increas the `version` patch number.
+- 1.6 Commit with the chore(release) commit type.
+- 1.7 Create a pull request from chore/release to develop.
```

Comment:
I know standard-version does say to use the `--follow-tags` flag in the output, but it's something I missed the first time doing this, so maybe that should be mentioned here in the process docs?